workspace_host/interface.js

```diff
@@ -244,7 +244,7 @@ async function _syncPushContainer(workspace_id, callback) {
         callback(null, workspace_id);
         return;
     }
-    var jobs_params = _recursiveUploadJobManager(workspaceName, workspaceName);
+    var jobs_params = _recursiveUploadJobManager(`/jobs/${workspaceName}`, workspaceName);
```

Comment:
@mwest1066 I just realized, we should only prepend `/jobs` if we have $HOST_JOBS_DIR, right?

Reply:
I modified it to this: https://github.com/PrairieLearn/PrairieLearn/pull/2723/commits/5f16e6a7ffad7222ef34bf38a5915f16d5950cc6

Reply:
Yep, that looks correct to me.